src/config.ts

```diff
@@ -78,9 +81,20 @@ export const SCENE_TILE_PROVIDERS = [
     maxZoom:  13,
     name: 'PlanetScope',
     prefix: 'planetscope',
+    catalogSection: 'planet', // Look in /planet/* in the catalog
     provider: 'Planet',
     url: `https://tiles{0-3}.${PLANET_BASE_URL}/v1/experimental/tiles/PSOrthoTile/__SCENE_ID__/{z}/{x}/{y}.png?api_key=__API_KEY__`,
   },
+  {
+    attributions: '<a href="https://docs.opendata.aws/landsat-pds/readme.html">Landsat PDS</a>',
+    maxZoom:  13,
+    name: 'Landsat8 (Public Data Set)',
+    prefix: 'landsat_localindex',
+    catalogSection: 'localindex', // Look in /localindex/* in the catalog
+    provider: 'Landsat PDS',
+    url: `${IMAGERY_ENDPOINT}/tiles/landsat/__SCENE_ID__/{z}/{x}/{y}.jpg`,
```

Comment:
I was trying to find in the api pull request where this url endpoint was setup but couldn't find it.  Will this work?

Reply:
https://github.com/venicegeo/bf-ia-broker/blob/398825acabd5b5b2c5d3a99286b976b1acf91745/landsat_localindex/handlers.go#L199-L260

With the endpoint registered here: https://github.com/venicegeo/bf-ia-broker/blob/398825acabd5b5b2c5d3a99286b976b1acf91745/cmd/bf-ia-broker/serve.go#L63

Reply:
To clarify, the fact that the URL is prefixed with `${IMAGERY_ENDPOINT}` means the full URL will look something like `bf-api.exampledomain/ia/tiles/landsat/<id>/Z/X/Y.jpg`, which hits the `/ia` API endpoint, which proxies to the IA broker. That's why my links to the code implementing this are in the IA broker, not the API.

Reply:
Ah that would explain why i didn't see it in the api pull request :)